### PR TITLE
Move parsing of cli arguments to function

### PR DESCRIPTION
**Pull request by mkosiarc**
Moved parsing of cli arguments to function for easier testing, also CHOICES is now a global variable, because it is used in a lot of places and it doesn't have to get passed to functions all the time.
@pbortlov @sidpremkumar please review

### CODE REVIEW

**Comment by mkosiarc** [comment]
@msimacek @amarza-rh could you please review

**Comment by pbortlov** [comment]
@mkosiarc this will be merge after #92

File: reviewrot/__init__.py [hunk 1]
```diff
@@ -177,17 +184,101 @@ def get_arguments(cli_arguments, config, choices):
...
+                           help='Path to CA certificate to use for SSL '
+                                'certificate verification')
+
+    parsed_args = parser.parse_args(args)
+    options = (
+        parsed_args.state,
+        parsed_args.value,
+        parsed_args.duration
+    )
+    if any(options) and not all(options):
```
Comment (pbortlov) [inline reviewrot/__init__.py]: @mkosiarc please add comment

File: reviewrot/__init__.py [hunk 2]
```diff
@@ -177,17 +184,101 @@ def get_arguments(cli_arguments, config, choices):
...
+        parser.error('Either no or all arguments are required')
```
Comment (pbortlov) [inline reviewrot/__init__.py]: @mkosiarc can you specify which arguments?

File: reviewrot/__init__.py [hunk 3]
```diff
@@ -177,17 +184,101 @@ def get_arguments(cli_arguments, config, choices):
     return parsed_arguments
 
 
-def is_valid_choice(argument, value, choices):
+def parse_cli_args(args):
+
+    parser = argparse.ArgumentParser(
+        description='Lists pull/merge/change requests for github, gitlab,'
+                    ' pagure and gerrit')
```
[the remark this answers is not in the document]
Comment (mkosiarc) [reply reviewrot/__init__.py]: I updated this part, and I will leave the rest for another PR, the task is mentioned in DEVOPSA-6079

**Comment by pbortlov** [comment]
I updated this part, and I will leave the rest for another PR, the task is mentioned in [DEVOPSA-6079](https://projects.engineering.redhat.com/browse/DEVOPSA-6079)

File: reviewrot/__init__.py (hunk 1 above)
Comment (pbortlov) [reply reviewrot/__init__.py]: This functionality will be modified in the next #93

File: reviewrot/__init__.py (hunk 2 above)
Comment (pbortlov) [reply reviewrot/__init__.py]: This functionality will be modified in the next #93

**Comment by pbortlov** [comment]
#83

**Pull request merged**